### ISSUE TITLE
How to loop over key/value pairs?

### ISSUE THREAD

**Issue by juliohm** [blackrock/NamedTuples.jl#42]
I am trying to use NamedTuples in my package, but couldn't find out how we are supposed to loop over key/value pairs, my attempt below failed:

```julia
nt = @NT(a=1, b=2.0)
for (k,v) in nt
  println(k, " = ", v)
end
```

**Comment by davidanthoff** [comment]
I believe a PR with a ``pairs`` function would be a nice addition to the package.

**Comment by juliohm** [comment]
Out of curiosity, will this be available in Julia v0.7? Should I close the issue?

**Comment by juliohm** [comment]
Since NamedTuples are gonna be part of the next version of the language, I am closing the issue. Hope the key/value pair loop is addressed there already.
